.travis.yml

```diff
@@ -3,12 +3,13 @@ cache: yarn
 script:
   - yarn --version
   - yarn setup
-  - yarn check
+  - yarn check --integrity
   - yarn lint
   - yarn test
```

Comment:
do we still need to run test and test:routes if coverage is doing it?

Reply:
Possibly not on travis.

Left `Test (unit)` and `Test (routes)` stages on Jenkins to have it visibly separated of what has failed especially that redoing this as a part of coverage gives just few extra secs to the run time, however I do not feel very strong about that.